aiida/restapi/translator/data/structure.py

```diff
@@ -41,31 +42,29 @@ def __init__(self, **kwargs):
         Initialise the parameters.
         Create the basic query_help
         """
-        super(StructureDataTranslator, self).__init__(Class=self.__class__,
-                                                      **kwargs)
-
+        super(StructureDataTranslator, self).__init__(Class=self.__class__, **kwargs)
 
     @staticmethod
-    def get_visualization_data(node, format=None):
+    def get_visualization_data(node, visformat=None):
```

Comment:
why not put `visformat="xsf"` here directly? then its clear by looking at the signature of the function what the default is